sys-power/apcupsd/files/apcupsd.powerfail.init-r1

```diff
@@ -9,7 +9,7 @@ depend() {
 }
 
 start() {
-	if [ "$(runlevel | cut -d' ' -f2)" = "0" -a -f /etc/apcupsd/powerfail ] ; then
+	if [ -a -f /etc/apcupsd/powerfail ] ; then
```

Comment:
The `-a` here means `and`, but now there's now previous condition.

It should just be `-f`.

Reply:
Thanks!